### PR TITLE
Took out map and compass from uncommon loot.

### PR DESCRIPTION
**Pull request by Rebel-Leader**

### CODE REVIEW

File: config/Wasteland/CityLoot.cfg [hunk 1]
```diff
@@ -80,8 +80,6 @@
         minecraft:bucket,1,1
         minecraft:bread,4,2
         minecraft:iron_ingot,2,1
-        minecraft:compass,1,1
-        minecraft:map,1,1
```
Comment (chiqors) [inline config/Wasteland/CityLoot.cfg:84]: why do you remove this?

Comment (Rebel-Leader) [reply config/Wasteland/CityLoot.cfg:84]: These items are pretty useless with Journeymap in game.

Comment (chiqors) [reply config/Wasteland/CityLoot.cfg:84]: well, i guess.. you are right.. well gonna accept this

**Review by chiqors — APPROVED** [review]
Done

**Pull request merged**